reference/toolstacks/python/README.md

```diff
@@ -0,0 +1,174 @@
+WSGI-based (Gunicorn / uWSGI) application server, Tornado, Twisted, or Python 3.5+ asyncio server.
+
+## Supported versions
+
+* 2.7
+* 3.5
+
+## WSGI-based configuration
+
+In this example, we use Gunicorn to run our WSGI application.
+Configure the `.platform.app.yaml` file with a few key settings
+as listed below, a complete example is included at the end of this section.
+
+1. Specify the `type` as `python:2.7` (other versions you may use are listed above).
+2. Build your application with the build hook.
+   Assuming you have your pip dependencies stored in `requirements.txt` and
+   a `setup.py` at the root of your application folder to execute build steps:
+
+   ```yaml
+   hooks:
+     build: |
+       pip install -r requirements.txt
+       pip install -e .
+       pip install gunicorn
+   ```
+
+   These are installed as global dependencies in your environment.
+
+3. Configure the command you use to start serving your application (this must
+   be a foreground-running process) under the `web` section, e.g.:
+
+   ```yaml
+   web:
+     commands:
+       start: "gunicorn -b $PORT project.wsgi:application"
+   ```
+
+   This assumes the WSGI file is `project/wsgi.py` and
+   the WSGI application object is named `application` in the WSGI file.
+
+4. Define the web locations your application is using:
+
+   ```yaml
+   web:
+     locations:
+       "/":
+         root: ""
+         passthru: true
+         allow: false
+       "/static":
+         root: "static/"
+         allow: true
+   ```
+
+   This configuration asks our web server to handle HTTP requests at "/static"
+   to serve static files stored in `/app/static/` folder
+   while everything else are forwarded to your application server.
+
+5. Create any Read/Write mounts. The root file system is read only.
+   You must explicitly describe writable mounts.
+
+   ```yaml
+   mounts:
+     "/tmp": "shared:files/tmp"
+     "/logs": "shared:files/logs"
+   ```
+
+   This setting allows your application writing files to `/app/tmp` and
+   have logs stored in `/app/logs`.
+
+Then, setup the routes to your application in `.platform/routes.yaml`.
+
+   ```yaml
+   "http://{default}/":
+     type: upstream
+     upstream: "app:http"
+
+   "https://{default}/":
+     type: upstream
+     upstream: "app:http"
+   ```
+
+Here is the complete `.platform.app.yaml` file:
+
+```yaml
+name: app
+type: python:2.7
+
+web:
+  commands:
+    start: "gunicorn -b $PORT project.wsgi:application"
+  locations:
+    "/":
+      root: ""
+      passthru: true
+      allow: false
+    "/static":
+      root: "static/"
+      allow: true
+
+hooks:
+  build: |
+    pip install -r requirements.txt
+    pip install -e .
+    pip install gunicorn
+
+mounts:
+  "/tmp": "shared:files/tmp"
+  "/logs": "shared:files/logs"
+
+disk: 512
+```
+
+## Using Python 3.5+ asyncio module
+
+The above Gunicorn based WSGI example can be modified to
+use Python 3.5+ asyncio module.
```

Comment:
`-` `use Python 3.5+ asyncio module.`
`+` `use the Python 3.5+ asyncio module.`